README.md

```diff
@@ -1 +1,12 @@
 # retail-url-scraper
+A web scraper that scrapes retail websites and returns URLs for all listed products. Runs on Express and Node
+
+To Install and run:
+
+1. npm install
+
+2. DEBUG=site-urls:* npm start
+
+
+node v10.16.3
```

Comment:
node is currently on +12 version - maybe you should update
If you switch to nvm, node updates will be much easier